package.json

```diff
@@ -12,6 +12,7 @@
     "build": "npx lerna run build",
     "build:changed": "npx lerna run build --since main",
     "test": "npm run ui-test --workspace=extester-test",
+    "coverage": "npm run test -- -- --coverage",
```

Comment:
could you define as `test:coverage` please? I feel it makes more sense in context it will run the tests before coverage is provided

Reply:
Yes, I will update the name.

Reply:
Updated in the latest commit.